File: Java/Account.java
public class Account {
    private Integer id;
    private String name;
    private String document;
    private String email;
    private String password;

    public Account(Integer id, String name) {
        this.id = id;
        this.name = name;
    }
    
    public Integer getId() {
        return id;
    }
    public void setId(Integer id) {
        this.id = id;
    }
    public String getName() {
        return name;
    }
    public void setName(String name) {
        this.name = name;
    }
    public String getDocument() {
        return document;
    }
    public void setDocument(String document) {
        this.document = document;
    }
    public String getEmail() {
        return email;
    }
    public void setEmail(String email) {
        this.email = email;
    }
    public String getPassword() {
        return password;
    }
    public void setPassword(String password) {
        this.password = password;
    }

    @Override
    public String toString() {
        return "Account [document=" + document + ", email=" + email + ", id=" + id + ", name=" + name + ", password="
                + password + "]";
    }

    
}

File: JS/Account.js
function Account (id, name) {
    this.id = id;
    this.name = name;
    this.document;
    this.email;
    this.password;
}

Account.prototype.toString = function(){
    console.log(this.id);
    console.log(this.name);
    console.log(this.document);
    console.log(this.email);
    console.log(this.password);
}